### DISCUSSION TITLE
"A new version of the protocol specification must be released every 6 weeks."?

### DISCUSSION THREAD

**Discussion by matthiasgeihs** [farcasterxyz/protocol#124] [General]
The [protocol spec](https://github.com/farcasterxyz/protocol/blob/main/docs/SPECIFICATION.md#5-versioning) states "A new version of the protocol specification must be released every 6 weeks."

At the same time, it's current version is `2023.7.12`.

The release cycle goal doesn't seem appropriate and should probably be removed?

**Comment by varunsrin** [comment]
we've been releasing new versions of the implementation every 6 weeks, the docs are just out of date
